src/NServiceBus.NHibernate.TransactionalSession.AcceptanceTests/TransactionSessionDefaultServer.cs

```diff
@@ -0,0 +1,25 @@
+namespace NServiceBus.TransactionalSession.AcceptanceTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Support;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+
+    public class TransactionSessionDefaultServer : DefaultServer
+    {
+        public override Task<EndpointConfiguration> GetConfiguration(RunDescriptor runDescriptor, EndpointCustomizationConfiguration endpointConfiguration,
+            Func<EndpointConfiguration, Task> configurationBuilderCustomization) =>
+            base.GetConfiguration(runDescriptor, endpointConfiguration, async configuration =>
+            {
+                // Explicitly enforcing the type to be scanned. Otherwise the scanner would not pick it up in the acceptance tests
+                endpointConfiguration.TypesToInclude.Add(typeof(TransactionalSession));
+                endpointConfiguration.TypesToInclude.Add(typeof(NHibernateTransactionalSession));
```

Comment:
I still think this shouldn't be required. If this is required with the latest version of NSB, I think we need to smoke test the actual behavior when disabling file scanning on a real endpoint.